.circleci/config.yml

```diff
@@ -1621,9 +1625,9 @@ jobs:
 
   # Layer 2: Builds.
   linux-x64-testing:
-    <<: *machine-linux-2xlarge
+    <<: *machine-linux-xlarge
     environment:
-      <<: *env-linux-2xlarge
+      <<: *env-global
```

Comment:
testing builds do not need the number of cores in `env-linux-2xlarge` because those builds always use goma.